HousingFinanceInterimApi.Tests/V1/UseCase/GenerateReportUseCaseTests.cs

```diff
@@ -3090,9 +3087,9 @@ public async Task GenerateReportUCStopsItsAttemptsToRetrieveTheUploadedOperating
             var stepResponse = await _classUnderTest.ExecuteAsync().ConfigureAwait(false);
 
             // assert
-            var secondsSpentInWaitForTheFile = (int) (lastAttempt - firstAttempt.Value).TotalSeconds + 1;
+            var secondsSpentInWaitForTheFile = (lastAttempt - firstAttempt).Value.TotalSeconds + (_retryInterval / 1000.0);
```

Comment:
What data type is the TodalSeconds?
Hopefully it's not an int as 200/1000=0.2 = 0 (int)